docs/community/communityguide.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Community Guides to Creating Synthetic Tokens on UMA
+sidebar_label: Community Guides to Creating Synthetic Tokens on UMA
+---
+
+We want to make the process of creating priceless contracts on UMA for synthetic tokens as easy as possible so that anyone is able to create one.  Community members have developed a wealth of resources and tools to make our ambition of Universal Market Access a reality.
+
+### Developing Your Idea
+
+Everything begins with an idea.  A great place to look at whether someone is already developing in the area that you are interested in, or to flesh out your own idea is the UMA Discourse [Ideas Bank](https://github.com/Madtodist/UMIPs/blob/master/UMIPs/Big%20Mac%20UMIPv1.md). 
+
+There you can get feedback from other community members around both the concept and technicalities of the derivative to allow you to start the process of building your synthetic asset and making it available for minting. 
+
+### Writing An UMIP
+
+A synthetic asset contract needs to know two things - the collateral type that the asset will be backed by, and the reference price of the synthetic asset. Note that “Price” in this context does not necessarily mean a monetary value, as synthetic assets can track anything.   
```

Comment:
Add at end of sentance

"Templates for UMIPs are now available on the [Discourse](https://discourse.umaproject.org/).  Starting a new topic in one of the UMIP template categories, will bring up a preformatted template to use as a base"